nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1,1001 +1,254 @@
 { config, lib, pkgs, ... }:
-
-with lib;
-
 let
+  inherit (lib) versionAtLeast mkEnableOption mkOption types mdDoc mkIf
+    optionals mkPackageOption mkRemovedOptionModule mkDefault
+    mkRenamedOptionModule literalExpression maintainers;
+
   cfg = config.services.hedgedoc;
 
   # 21.03 will not be an official release - it was instead 21.05.  This
   # versionAtLeast statement remains set to 21.03 for backwards compatibility.
   # See https://github.com/NixOS/nixpkgs/pull/108899 and
   # https://github.com/NixOS/rfcs/blob/master/rfcs/0080-nixos-release-schedule.md.
-  name = if versionAtLeast config.system.stateVersion "21.03"
+  name =
+    if versionAtLeast config.system.stateVersion "21.03"
     then "hedgedoc"
     else "codimd";
```

Comment:
```suggestion
  name = if versionAtLeast config.system.stateVersion "21.03" then
    "hedgedoc"
   else
     "codimd";
```

Reply:
This is interesting. I used `nixpkgs-fmt` on the code before making this PR. Are there other style guides or tools which are better?

Reply:
Also, what's the deal with the `else` being indented 3 spaces for 2 space indendation? Is it like semi-indented, but just not quite?

Reply:
I've updated this, but I'm leaving it open out of curiosity

Reply:
> Also, what's the deal with the `else` being indented 3 spaces for 2 space indendation? Is it like semi-indented, but just not quite?

this is me eyeballing wrong

> This is interesting. I used `nixpkgs-fmt` on the code before making this PR. Are there other style guides or tools which are better?

I usally use nixpkgs-fmt but remove the three nonsensical formattings it does

Reply:
What do you mean by "three nonsensical formattings"?